db/migrate/20230329161740_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```

Comment:
- Hey, the text should be a text `t.text :text` and not a string. please, let's fix this.